lib/syskit/network_generation/engine.rb

```diff
@@ -537,7 +537,8 @@ def reconfigure_tasks_on_static_port_modification(deployed_tasks)
                     work_plan
                     .find_tasks(Syskit::TaskContext).not_finished.not_finishing
```

Comment:
You should reject explicitly all read-only tasks here. This involves configuring, and that's something we don't do for them.

A.k.a. add `.find_all { |t| !t.read_only? }` before the last `find_all`